### PR TITLE
Change style edit button

### LINKED ISSUE #215: Edit button doesn't look like the mockups
In FORD3:
Screen Shot 2019-04-11 at 15 59 41

In the mockups:
Screen Shot 2019-04-11 at 16 00 36

### PR DESCRIPTION
**Pull request by boney-bun**
fix #215

### CODE REVIEW

File: django_project/ford3/static/css/provider_form.css [hunk 1]
```diff
@@ -127,3 +127,16 @@ ul.errorlist {
     padding-inline-start: 10px;
     color: red;
 }
+
+.edit-campus {
+    font-family: "Lato Black";
```
Comment (pierrealixt) [inline django_project/ford3/static/css/provider_form.css]: this should be set at a higher level.
maybe in `html`

File: django_project/ford3/static/css/provider_form.css [hunk 2]
```diff
@@ -127,3 +127,16 @@ ul.errorlist {
...
+    color: #2a3056;
```
Comment (pierrealixt) [inline django_project/ford3/static/css/provider_form.css]: please use CSS variables.
See the top of `stylesheet.css`

**Comment by pierrealixt** [comment]
@boney-bun Add a gif please.

**Comment by boney-bun** [comment]
the new style:
ford3#237

**Comment by pierrealixt** [comment]
@boney-bun could you add a screenshot/gif?

**Comment by boney-bun** [comment]
@pierrealixt below is the screen shot:
Screen Shot 2019-04-22 at 11 11 29 AM

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**